packages/toggle-core/src/Write/Payload.php

```diff
@@ -36,6 +45,6 @@ public static function fromJsonString(string $jsonPayload): self
      */
     public function data(): array
```

Comment:
Maybe we could rename the getter to `criteria` as the class property:
```suggestion
    public function criteria(): array
```

Reply:
Nice catch ;-D done.